README.md

```diff
@@ -673,6 +674,41 @@ Additional options (configurable via `ConnectWithOpts`):
 * `ClusterDiscoveryTime` - time interval to ask server for updated address list (works on with `NodesGetFunctionName` set)
 * `NodesGetFunctionName` - server lua function name to call for getting address list
 
+## Tests
+
+You need to [install Tarantool](https://www.tarantool.io/en/download/) to run tests.
+See [Installation](#installation) section for requirements.
+
+To install test dependencies (like [tarantool/queue](https://github.com/tarantool/queue) module), run
+```bash
+./deps.sh
+```
+
+To run tests, call
+```bash
+go clean -testcache && go test ./... -v -p 1
+```
+Tests set up all required `tarantool` processes before run.
+
+To run subpackage tests, go to a subpackage folder and run the same command.
+```bash
+cd multi
+go clean -testcache && go test -v
+```
+
+Queue tests require [tarantool/queue](https://github.com/tarantool/queue) module, install it with `./deps.sh`.
+```bash
+cd queue
+./deps.sh
+go clean -testcache && go test -v
+```
+
+UUID tests require [Tarantool 2.4.1 or newer](https://github.com/tarantool/tarantool/commit/d68fc29246714eee505bc9bbcd84a02de17972c5).
```

Comment:
Is there a way to run all tests at once instead of running tests for each module one by one? I bet it would be more convenient.

Reply:
It is `go clean -testcache && go test ./... -v -p 1`, I'll rewrite text to be more obvious

Reply:
Fixed